### PR TITLE
Migrate CI from Travis to Circle

### LINKED ISSUE #385: Migrate to CircleCI
To finish the CKEditor 5 migration, we need to configure the CI.

* We do not need dynamic configs like we did in CKEditor 5.
* Coveralls need the `COVERALLS_REPO_TOKEN` environment variable.
* [The Slack notifier](https://github.com/ckeditor/ckeditor5-dev/blob/9d3a9cfc58fe04035b33edd4b4de380f16d5ec5f/packages/ckeditor5-dev-ci/bin/notify-circle-status.js) needs the `CKE5_GITHUB_TOKEN`, `CKE5_CIRCLE_TOKEN`, and `CKE5_SLACK_WEBHOOK_URL` variables.
* To install the latest Chrome, use the [`ckeditor5-dev-ci-install-latest-chrome`](https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-dev-ci/package.json#L29C6-L29C44) binary.
* To figure out steps to execute, follow the Travis configuration.
* When everything works as expected, remove Travis files.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Migrate CI from Travis to Circle. Closes #385.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ module.exports = function( config ) {
 					dir: './coverage',
 					subdir: '.',
 					type: 'lcovonly'
+				},
+				{
+					dir: './coverage',
+					subdir: '.',
+					type: 'json'
```
Comment (pomek) [inline karma.conf.js:47]: JSON file is required for `nyc`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+#            #!/bin/bash
+#
+#            # Create the `nyc` directory.
+#            mkdir .nyc_output
+#
+#            # Copy results.
+#            cp coverage/coverage-final.json .nyc_output
+#
+#            # Verify results.
+#            npx nyc check-coverage --branches 100 --functions 100 --lines 100 --statements 100
```
Comment (pomek) [inline .circleci/config.yml:89]:
```suggestion
      - run:
          name: Verify the code coverage
          command: |
            #!/bin/bash

            # Create the `nyc` directory.
            mkdir .nyc_output

            # Copy results.
            cp coverage/coverage-final.json .nyc_output

            # Verify results.
            npx nyc check-coverage --branches 100 --functions 100 --lines 100 --statements 100
```

Please apply the suggestion if https://github.com/ckeditor/ckeditor5-angular/pull/387 is merged before this PR.

**Review by psmyrek — COMMENTED** [review]
Two things to align:
1. Build status badge still asks Travis.
2. There is `CHROME_TRAVIS_CI` env variable in Karma configuration.

**Comment by pomek** [comment]
> Build status badge still asks Travis.

image

Where? :D

**Pull request merged**